src/components/AccountContainer.js

```diff
@@ -13,48 +13,33 @@ class AccountContainer extends Component {
 
     this.state = {
       searchTerm: '',
-      transactions: [
-        {
-          id: 1,
-          posted_at: "2017-02-28 11:00:00",
-          description: "Leather Pants, Gap co.",
-          category: "Fashion",
-          amount: -20000
-        },
-        {
-          id: 2,
-          posted_at: "2017-02-29 10:30:00",
-          description: "Paycheck from Bob's Burgers",
-          category: "Income",
-          amount: 100000
-        },
-        {
-          id: 3,
-          posted_at: "2017-05-24 10:53:00",
-          description: "'Pair Programming Illuminated' by Laurie Williams and Robert Kessler",
-          category: "Book",
-          amount: 1498
-        },
-        {
-          id: 4,
-          posted_at: "2017-05-24 08:52:00",
-          description: "Medium Iced Cold Brew, Gregory's Coffee",
-          category: "Coffee",
-          amount: 365
-        }
-      ]
+      transactions: []
     }
+    this.handleChange = this.handleChange.bind(this)
   }
 
   handleChange(event) {
-    // your code here
+    let searchTerm = event.target.value
+    this.setState({
+      searchTerm
+    })
+  }
+
+  componentDidMount(){
+    fetch("https://boiling-brook-94902.herokuapp.com/transactions")
+    .then((resp) => resp.json())
+    .then((transactions) => {
+      this.setState({
+        transactions
+      })
+    })
   }
 
   render() {
 
     return (
       <div>
-        <Search searchTerm={this.state.searchTerm} handleChange={"...add code here..."} />
+        <Search searchTerm={this.state.searchTerm} handleChange={(text) => this.handleChange(text)} />
```

Comment:
you should be able to just pass `this.handleChange` - no need to wrap it in this anonymous function. Also, what you're calling `text` here is actually the event I believe.